src/test/kotlin/com/expedia/graphql/hooks/DataFetcherExecutionPredicateTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.expedia.graphql.hooks
+
+import com.expedia.graphql.exceptions.GraphQLKotlinException
+import graphql.schema.DataFetchingEnvironment
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.spyk
+import io.mockk.verify
+import org.junit.jupiter.api.Test
+import kotlin.reflect.KParameter
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+internal class DataFetcherExecutionPredicateTest {
```

Comment:
depending on https://github.com/ExpediaDotCom/graphql-kotlin/issues/134 this may need to be completely rewritten

Reply:
@dkuc84 Actually these tests could just be deleted since we don't need to test an interface and we have implementation tests here already: https://github.com/ExpediaDotCom/graphql-kotlin/blob/ce018a9876c2de503b3847043992d71c70bf19cd/src/test/kotlin/com/expedia/graphql/KotlinDataFetcherTest.kt#L54

Reply:
The other hook tests would still be valid though

Reply:
You can see the changes required to fix #134 here: https://github.com/smyrick/graphql-kotlin/compare/hooks-test...smyrick:datafetcher-predicate?expand=1

Reply:
lets drop this test then for now, we can reintroduce it later if needed

Reply:
Done